qiskit_experiments/library/characterization/t1.py

```diff
@@ -131,3 +132,22 @@ def circuits(self, backend: Optional[Backend] = None) -> List[QuantumCircuit]:
             circuits.append(circ)
 
         return circuits
+
+    def _pre_transpile_hook(self, backend: Backend):
+        """Set timing constraints if backend is real hardware."""
+
+        if not backend.configuration().simulator and not isinstance(backend, FakeBackend):
+            timing_constraints = getattr(self.transpile_options.__dict__, "timing_constraints", {})
+
+            # alignment=16 is IBM standard. Will be soon provided by IBM providers.
+            # Then, this configuration can be removed.
+            timing_constraints["acquire_alignment"] = getattr(
+                timing_constraints, "acquire_alignment", 16
+            )
```

Comment:
shouldn't something like this be in the default transpile options of the class? Even better would be if the backend were to report this but that is out of scope for this PR.

Reply:
Can default option depend on the backend?

Reply:
In principle this depends on the backend. IBM backends currently use standardized control electronics for every backend, and the alignment factor doesn't change. Of course this depends on the provider. Currently, we are still waiting our provider to report these values. 

Note that, if we give a default alignment value, we may accidentally override the system reported value. This may cause a silent bug if these alignment values are inconsistent. Of course this hard-coded value may cause the problem to other providers, but I don't think giving a default value is not good idea.